packages/react-components/react-swatch-picker-preview/src/utils/contrastUtils.ts

```diff
@@ -0,0 +1,108 @@
+export type Rgb = [number, number, number] | null;
```

Comment:
Maybe it would be better to use existing library. But if not, I'll need to add a case when hex color is "XXX" and when color is written by name.

Reply:
There is a very well tested library called [Polished](https://polished.js.org) that can do a lot of the calculations done in this file.

For example: [parseToRgb](https://polished.js.org/docs/#parsetorgb) and [getContrast](https://polished.js.org/docs/#getcontrast) could be used here.

Reply:
Yeah, this one is good. I'll add it to best practices as a suggestion

Reply:
This is outside of the scope of this PR of course and does not necessarily need to be added, but one thing I can think as a solution could be to create a story or a CodeSandbox/Stackblitz example on how to handle this in the userland, either by using this library or something else (your utils functions for example), so users can know how to apply the best practices.